### PR TITLE
namcap PKGBUILD

### PR DESCRIPTION
**Pull request by casz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,9 @@ services:
 
 arch:
   script:
+    - "pacman -S namcap --noconfirm"
```
Comment (Azd325) [inline .travis.yml]: I think you can install this package over the package section. 
https://github.com/mikkeloscar/arch-travis

Comment (casz) [reply .travis.yml]: ya, I forgot 😅

**Comment by casz** [comment]
Feel free to squash when it is green 😄

**Comment by MarcinWieczorek** [comment]
You can namcap the *.pkg.tar.xz too

**Comment by Azd325** [comment]
> You can namcap the *.pkg.tar.xz too

Should be the same
https://wiki.archlinux.org/index.php/namcap#How_to_use_it

**Comment by Azd325** [comment]
> > You can namcap the *.pkg.tar.xz too
> 
> Should be the same
> https://wiki.archlinux.org/index.php/namcap#How_to_use_it

Ok It not the same :) 
>namcap is a package analysis utility that looks for problems with Arch Linux packages or their PKGBUILD files. It can apply rules to the file list, the files themselves, or individual PKGBUILD files. https://jlk.fjfi.cvut.cz/arch/manpages/man/namcap.1

**Comment by casz** [comment]
Are you okay with less compression, it speeds up build significantly.

Before: Ran for 4 min 47 sec
After: Ran for 2 min 58 sec

**Comment by Azd325** [comment]
I think this is fine 👍

**Comment by casz** [comment]
🙌

**Comment by Azd325** [comment]
Merged in b7ca267ba0e9af7e5982153f8b2ca38ccbdf9917 8ca41e561530b3f1be236c73a70aa0c728e345e9 8a324d578a9e450c5dd237c521ac7928919fb1c7

**Pull request closed without merging**